meinberlin/apps/contrib/templates/meinberlin_contrib/components/alert.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n %}
+
+<div class="alert alert--{{ alert_type }}" 
+role="alert" 
+aria-live="{% if alert_type == 'danger' %}assertive{% else %}polite{% endif %}" 
+aria-describedby="{% if alert_headline %}alert__headline{% endif %} alert__text"
+aria-atomic="true"
+>
+  <div class="container">

```

Comment:
btw, we can drop the container class as it has no effect (bootstrap lgeacy)

Reply:
can I also drop it on a4?

Reply:
no, in a4 we still need for the other projects